specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-10-03-preview/HybridCompute.json

```diff
@@ -2972,13 +3336,112 @@
           "description": "The resource id of the license."
         }
       },
-      "description": "Describes the Update properties of a License Profile."
+      "description": "Describes the Update properties of a ESU License Profile."
+    },
+    "LicenseProfileSubscriptionStatusUpdate": {
+      "type": "string",
+      "enum": [
+        "Enable",
+        "Disable"
+      ],
+      "x-ms-enum": {
+        "name": "LicenseProfileSubscriptionStatusUpdate",
+        "modelAsString": true
+      },
+      "description": "Indicates the new subscription status of the Windows Server, IoT Enterprise or Product Features."
+    },
+    "ProductFeatureUpdate": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Product feature name."
+        },
+        "subscriptionStatus": {
+          "$ref": "#/definitions/LicenseProfileSubscriptionStatusUpdate",
+          "description": "Indicates the new status of the product feature."
+        }
+      },
+      "description": "Product Feature"
+    },
+    "WindowsServerProfileUpdateProperties": {
```

Comment:
WindowServerProfile and IoTEnterpriseProfile, here and below, seem to share the same properties (assuming my eyes aren't playing tricks on me). 

Have you considered having a single <Something>Profile with a Type property that could be WindowsSever or IoTEnterprise, currently?

Reply:
Combined these two and added osType with Enum values(**WindowsServer** AND **WindowsIoTEnterprise**)